R/ui_helpers.R

```diff
@@ -664,3 +672,38 @@ get_url_parameters <- function(session) {
 
     parameters
 }
+
+
+# Override bs4Dash tooltip function to allow html handling in tooltips
+tooltip <- function(tag, title, placement = c("top", "bottom", "left", "right")) {
```

Comment:
just for my information how is our tooltip different from the bs4dash tooltip?

Reply:
bs4Dash does not allow html tags inside a directly added tooltip in the UI (while they allow it if you added the tooltip programmatically!!!). So, we just allowed using html tags for directly created tooltip tags